scripts/edit-logs.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+path=''
+directory='false'
+
+print_usage() {
+  echo "Sctipt is to help replace go-ns logger to log.go logger. This script will update the package in use and attempt to refactor current logs into the new structure."
+  echo "This script cannot handlelogs across multiple lines."
+  echo "Usage: ./edit-logs.sh -d -p /go-projects/search-api/handlers"
+}
+
+while getopts 'hdp:' flag; do
+  case "${flag}" in
+    p) path="${OPTARG}"
+    ;;
+    d) directory='true'
+    ;;
+    h)
+      echo ""
+      echo "Update Logger Script"
+      echo ""
+      echo " * Updates logging library in file or directory"
+      echo " * Attempts to update as many variations of old logs to new logs"
+      echo " * Script is unable to update logs which traverse multiple lines"
+      echo ""
+      echo "Options are:"
+      echo ""
+      echo "      OPTION    DESCRIPTION                                 EXAMPLE ('' does NOT indicate default value)"
+      echo "        -p      The path to file or directory from \$HOME    \"go/src/github.com/myNewService\""
+      echo "        -d      Boolean flag, set if path is to directory   No value is needed, it defaults to false if not set"
+      echo "        -h      Help                                        No value is needed, returns description of flags"
+      echo ""
+      exit 0
+    ;;
+    \?)
+      echo "ERROR: Unknown option $OPTARG"
+    ;;
+    *) print_usage
+       exit 1
+       ;;
+  esac
+done
+
+home="$HOME"
+
+path=${path:?ERROR: var not set [-p path]}
```

Comment:
I wouldn't use this (seldom-used!) erroring. I'd replace as below (or put an `else` in the `if ... $path` on line 75)
```suggestion
if [[ -z $path ]]; then echo "ERROR: var not set [-p path]" >&2; exit 1; fi
```